src/test/java/com/exasol/adapter/dialects/generic/GenericSqlDialectTest.java

```diff
@@ -15,45 +14,48 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 
 import com.exasol.adapter.AdapterProperties;
-import com.exasol.adapter.dialects.PropertyValidationException;
-import com.exasol.adapter.dialects.SqlDialect;
+import com.exasol.adapter.dialects.*;
+import com.exasol.adapter.jdbc.ConnectionFactory;
 
 @ExtendWith(MockitoExtension.class)
 class GenericSqlDialectTest {
-    private Map<String, String> rawProperties;
     @Mock
-    private Connection connectionMock;
-    @Mock
-    private DatabaseMetaData metadataMock;
+    private ConnectionFactory connectionFactoryMock;
 
     @BeforeEach
-    void beforeEach() throws SQLException {
-        this.rawProperties = new HashMap<>();
-        when(this.connectionMock.getMetaData()).thenReturn(this.metadataMock);
-        when(this.metadataMock.supportsMixedCaseIdentifiers()).thenReturn(true);
-        when(this.metadataMock.supportsMixedCaseQuotedIdentifiers()).thenReturn(true);
+    void beforeEach(@Mock final DatabaseMetaData metadataMock, @Mock final Connection connectionMock)
+            throws SQLException {
+        when(metadataMock.supportsMixedCaseIdentifiers()).thenReturn(true);
+        when(metadataMock.supportsMixedCaseQuotedIdentifiers()).thenReturn(true);
+        when(connectionMock.getMetaData()).thenReturn(metadataMock);
+        when(this.connectionFactoryMock.getConnection()).thenReturn(connectionMock);
     }
 
     @Test
     void testValidateCatalogProperty() throws PropertyValidationException {
-        setMandatoryProperties("GENERIC");
-        this.rawProperties.put(CATALOG_NAME_PROPERTY, "MY_CATALOG");
-        final AdapterProperties adapterProperties = new AdapterProperties(this.rawProperties);
-        final SqlDialect sqlDialect = new GenericSqlDialect(this.connectionMock, adapterProperties);
-        sqlDialect.validateProperties();
+        final Map<String, String> rawProperties = Map.of( //
+                CATALOG_NAME_PROPERTY, "MY_CATALOG", //
```

Comment:
Make it also qualified as two below or make all static import

Reply:
Fixed.